runtime/src/main/java/org/corfudb/runtime/view/AbstractView.java

```diff
@@ -89,7 +88,12 @@ private Layout getLayoutUninterruptibly() {
                 log.error("getLayoutUninterruptibly: Encountered error. Aborting layoutHelper", ex);
                 throw (Error) ex.getCause();
             }
-            if (ex.getCause() instanceof RuntimeException) {
+            if (ex.getCause() instanceof WrongClusterException) {
+                log.warn("layoutHelper: Cluster reconfiguration or incorrect cluster", ex.getCause());
```

Comment:
aren't you missing a placeholder in the logger?

Reply:
I think it is good without the `{}`.  Since we are returning the throwable, it will print the stack trace of the exception. You had done it in L156, I copied it from there. Thanks! :)

More info: https://stackoverflow.com/a/6374166/5213405

Reply:
I always thought that applied to `log.error` and not `log.warn` but maybe I am wrong.